scripts/benchmark_multithreaded.py

```diff
@@ -28,7 +28,7 @@
 # python3 ./scripts/benchmark_multithreaded.py -e ./build-release/hyriseBenchmarkTPCH --runs 1000 -v --scale 1 --cores 10 20 30 --queries 1,3,6,12 --clients 20
 
 
-MAX_CORE_COUNT = multiprocessing.cpu_count()
```

Comment:
Thanks for your changes! I agree with everything except this one. The Topology already handles this if we are on a non-NUMA system. On a NUMA-system it uses all available CPUs. I don't think we should restrict this in two different spots, if it has to go through one of them anyway. Also this would easily get overridden when specifying --cores or --max-cores, while still possibly being changed by the Topology.